docs/books/user-guide/routing.adoc

```diff
@@ -551,6 +551,87 @@ autoLink {
 For information about additional attributes, see link:{qdrouterdConfManPageUrl}#_autolink[autoLink] in the `qdrouterd.conf` man page.
 --
 
+=== Handling Undeliverable Messages for an Address
+
+You handle undeliverable messages for an address by configuring _fallback destinations_.
+A fallback destination is a queue on a broker that stores messages that are not directly routable to any consumers.
+
+During normal message delivery,
+{RouterName} delivers messages to the consumers that are attached to the router network.
+However, if no consumers are reachable,
+the messages are diverted to any fallback destinations that were configured for the address.
```

Comment:
... any fallback destinations that are attached for the address...
It's possible that the auto-links are inactive, in which case messages aren't delivered anywhere.  Also, it is possible for a fallback destination to attach by itself (using the qd.fallback capability) without using an auto-link.